libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/component/camera/CameraViewModel.kt

```diff
@@ -21,7 +21,7 @@ sealed class CameraAction {
 }
 
 class CameraViewModel(
-    private val locationBehavior: LocationBehavior,
+    private val locationViewModel: LocationViewModel,
```

Comment:
Instead of injecting `LocationViewModel` into `CameraViewModel`, why not inject `LocationViewModel` into `CameraComponent` and let the component ask for location. I don't think we should store current location in the camera since it is available via `LocationViewModel`

Reply:
Sounds good to me!